### DISCUSSION TITLE
Variables with "." in their name.

### DISCUSSION THREAD

**Discussion by jamestjsp** [FreeOpcUa/opcua-asyncio#1591] [Q&A]
What is the beast way to deal with variables with period in their names?

This works fine 
 ```
       var = await client.nodes.root.get_child('0:Objects/2:DA/2:Analog/2:TI-1001/2:Measurement')
 ```
But this!!
 ```
        var = await client.nodes.root.get_child('0:Objects/2:DA/2:Analog/2:AI0001.PV/2:Measurement')
```
I believe the issue is caused by the *.PV.

**Comment by schroeder-** [comment]
We are using RelativePaths here, you have to escape a "." dot with a "&". 
So this should work:
`var = await client.nodes.root.get_child('0:Objects/2:DA/2:Analog/2:AI0001&.PV/2:Measurement')`
Or you switch to a list of browse names:
`var = await client.nodes.root.get_child(['0:Objects','2:DA','2:Analog', '2:AI0001.PV', '2:Measurement')`

**Reply by jamestjsp** [reply]
That works :)
